backstop_data/engine_scripts/puppet/page-objects/crm-page.js

```diff
@@ -98,6 +100,24 @@ module.exports = class CrmPage {
     await Promise.all([
       this.engine.click(selector),
       this.engine.waitForNavigation()
-    ]);      
+    ]);
+  }
+
+  /**
+   * Checks if there are no price fields present on the page or not
+   * 
+   * @param {String} type - the title of the message to be displayed
+   */
+  async checkIfPriceFieldsAreEmpty (type) {
```

Comment:
this is too specific for the generic `CrmPage` object. Create a more specific page (`PriceSetPage` ?) which will extend `CrmPage` and in which you will place this method